operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ServiceOperator.java

```diff
@@ -70,6 +77,31 @@ public ServiceOperator(Vertx vertx, KubernetesClient client) {
         }
     }
 
+    /**
+     * Finds annotations managed by the Rancher cattle agents (if any), and merge them into the desired spec.
+     *
+     * This makes sure there is no infinite loop where Rancher tries to add annotations, while Rancher keeps
+     * removing them during reconciliation.
+     *
+     * @param current   Current Service
+     * @param desired   Desired Service
+     */
+    private void patchCattleAnnotations(Service current, Service desired) {
```

Comment:
instead of having a specific method for just this "cattle" annotation, why not having a more general one passing a list of annotations to patch which in this case will continue just one annotation for now?

Reply:
Is there a preferred place (like configuration file / dependency) to store this "list of annotations", or would the `private static final` OK for this?

Reply:
We have this `Annotations` class https://github.com/strimzi/strimzi-kafka-operator/blob/master/operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

Reply:
I have refactored the code; can you check if this is better? :)